climsight.py

```diff
@@ -443,5 +676,6 @@ def extract_climate_data(lat, lon, _hist, _future):
             future_uas_str=data_dict["future_uas"],
             hist_vas_str=data_dict["hist_vas"],
             future_vas_str=data_dict["future_vas"],
+            nat_hazards = filtered_events,
```

Comment:
The same problem as before - only part of the table get into the final prompt. I would think of a summary.